dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5097,7 +5097,13 @@ public String toString() {
               + "Set value less than or equal to 0 to disable rate limits.")
           .setDefaultValue(0)
           .setScope(Scope.SERVER)
-
+          .build();
+  public static final PropertyKey PROXY_S3_TRANSFER_BUFFER_BYTES =
```

Comment:
Can you rename it to be PROXY_S3_TRANSFER_BUFFER_BYTES -> PROXY_S3_TRANSFER_BUFFER_SIZE?
as it would align with what you saw in jetty?

Reply:
done